cpp/tests/reductions/reduction_tests.cu

```diff
@@ -684,10 +687,19 @@ struct StringReductionTest : public cudf::test::BaseFixture {
 };
 
 // ------------------------------------------------------------------------
-TEST_F(StringReductionTest, MinMax)
+std::vector<std::string> string_list[] = {
```

Comment:
Can you add a test case where the column contains a string with the sentinel value to ensure it works correctly?

Reply:
testcase added.